tiltak/src/main/kotlin/no/nav/amt/tiltak/deltaker/service/BrukerServiceImpl.kt

```diff
@@ -46,9 +43,8 @@ class BrukerServiceImpl(
 		brukerRepository.slettBruker(personIdent)
 	}
 
-	override fun getOrCreate(fodselsnummer: String): UUID {
-		val bruker = brukerRepository.get(fodselsnummer) ?: createBruker(fodselsnummer)
-		return bruker.id
+	override fun getIdOrCreate(fodselsnummer: String): UUID {
+		return brukerRepository.get(fodselsnummer)?.id ?: createBruker(fodselsnummer)
```

Comment:
Er det ikke like greit å returnere hele objektet iom at man henter det allerede så blir det litt mer fleksibelt for fremtiden?

Reply:
Eller, ser nå at det allerede var en UUID som ble returnert